tests/test_model_import_and_export.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from capymoa.misc import save_model, load_model
+from capymoa.classifier import AdaptiveRandomForestClassifier
+from capymoa.datasets import ElectricityTiny
+from capymoa.evaluation import prequential_evaluation
+import os
+
+
+def test_save_and_load_model_with_valid_model():
+    # Arrange
```

Comment:
I don't get these comments, # Arrange, # Act, # Assert. Can we just remove them?